web-client/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,23 @@
+<ion-toolbar>
+  <ion-buttons slot="start">
+    <ion-button>
+      <ion-icon name="menu" src="icon-only"></ion-icon>
+    </ion-button>
+  </ion-buttons>
+  <ion-grid>
+    <ion-row class="ion-align-items-center">
+      <ion-col size="auto">
+        <img src="assets/img/Nautilus.svg" />
```

Comment:
It's probably better to import and use the Nasalisation font for this, rather than SVG?

Reply:
The title isn't using Nasalisation. it's a custom vector

Reply:
Ah, did Abby draw it from scratch, or is it a vector export of some other source?

Reply:
I think she did, I just exported from the design.

Reply:
Okay, I checked with Abby: she says it's actually this font:

- https://fonts.google.com/specimen/Audiowide

Can we import and use that?

Reply:
I imported it in #31 and replaced the image with text.